### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by pietern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+```bash
+brew update --force
+```
+
+## Installing an older version
+
+If you need to install an older version of the Databricks CLI, you can do so with:
+
+```bash
+brew tap-new databricks/tap-old
```
Comment (mgyucht) [inline README.md:35]: Is this a normal SOP for Homebrew tap users? It seems very cumbersome.

**Review by mgyucht — APPROVED** [review]
Overall, LGTM. Non-blocking question on the way that tapping works for older versions of the CLI.

File: README.md (hunk 1 above)
Comment (pietern) [reply README.md:35]: Yes, it is not possible to extract into the same tap where it is defined:

```
% brew extract --version=0.218.0 databricks/tap/databricks databricks/tap
Error: Cannot extract formula to the same tap!
```

Comment (pietern) [reply README.md:35]: And this is really a workaround -- Homebrew has first-class support for multiple versions of the same tap as long as you keep all versions as a separate file in the "Formula" directory. I don't think that's worth it for us (yet?).

**Pull request merged**